tutorials/kitties-part-01/0-intro.mdx

```diff
@@ -0,0 +1,56 @@
+---
+tags:
+  - pallet
+  - polkadot.js
+keywords: [pallet, polkadot-js]
+description:
+  From the substrate node template to a fully functional blockchain that handles kitties NFTs.
+level: intermediate
+duration: 5h
+---
+
+# How to build the Substrate Kitties Chain
+
+![kitties](./assets/kitties.png)
+
+Welcome to the Substrate Kitties tutorial part 1. Designed to teach how to build a blockchain that handles 
+the creation and ownership management of NFT Kitties.
+
+It will cover various aspects of building the Kitties pallet, including creation, breeding, 
+transfer, price setting, and purchasing kitties. Additionally, it will provide guidance on developing the 
+front-end UI for interaction with the chain.
+
+This tutorial is an updated/adapted version of two previous tutorials, the [CryptoKitties on Substrate](https://www.shawntabrizi.com/substrate/cryptokitties-on-substrate/)
+and the [Build the Substrate Kitties Chain](https://doc.deepernetwork.org/tutorials/v3/kitties/pt1/). The current tutorial uses the polkadot-v0.9.39 version of
+the Substrate Node Template. It was also added some extra sections exploring the node template and the polkadot-js javascript console.
+
+## What we will build?
+
+A substrate based blockchain for Kitties that can do the following:
+
+- Be created either by some original source or by being breed using existing Kitties.
```

Comment:
What do you mean by "original source" maybe rephrase?

Id put the enumeration in different format so it doesnt start with "Be" but rather

...based blockchain for Kitties, where you can:

- create Kitties ...
- sell kitties at a price ...
- transfer Kitties ...